src/Estimators/EstimatorManagerCrowd.cpp

```diff
@@ -31,10 +31,23 @@ EstimatorManagerCrowd::EstimatorManagerCrowd(EstimatorManagerNew& em)
   for (int i = 0; i < em.Estimators.size(); i++)
     scalar_estimators_.push_back(em.Estimators[i]->clone());
   MainEstimator = scalar_estimators_[EstimatorMap[MainEstimatorName]];
-  if (em.Collectables)
-    Collectables = em.Collectables->clone();
+  std::for_each(em.operator_ests_.begin(), em.operator_ests_.end(),
```

Comment:
Really don't like `begin/end` and range-based loops are better alternatives without the need of any lambda.